pennylane_lightning/src/bindings/Bindings.cpp

```diff
@@ -324,6 +324,29 @@ void lightning_class_bindings(py::module &m) {
                 const std::vector<size_t> &wires) {
                  return M.expval(operation, wires);
              })
+        .def("generate_samples",
+             [](Measures<PrecisionT> &M, size_t num_shots) {
+                 auto &&result = M.generate_samples(num_shots);
+                 // int num_samples = num_shots;
+                 // std::vector<int> result(num_samples*wires.size(),0);
+
+                 int num_wires = result.size() / num_shots;
+                 ssize_t ndim = 2;
```

Comment:
I think same here, if we don't need it signed. Though, ignore if this is a requirement from the binding interface.
```suggestion
                 const size_t ndim = 2;
```